src/interpolators.jl

```diff
@@ -128,6 +133,85 @@ function (sim::ScatteredInterpolationMethod)(
 
 end
 
+"""
+    NaturalNeighboursMethod(; method=Sibson(1), kwargs...)
+
+Interpolator that uses the [NaturalNeighbours.jl](https://github.com/DanielVandH/NaturalNeighbours.jl) package
+to interpolate the data.  This uses Delaunay triangulations and 
+the corresponding Voronoi diagram to interpolate the data, and offers
+a variety of methods like `Sibson(::Int)`, `Nearest()`, and `Triangle()`.
+
+The advantage of Voronoi-diagram based methods is that they are more robust to 
+irregularly distributed datasets and some discontinuities, which may throw off 
+some polynomial based methods as well as independant distance weighting (kriging). 
+See [this Discourse post](https://discourse.julialang.org/t/ann-naturalneighbours-jl-natural-neighbour-interpolation-and-derivative-generation/99164/11) for more information on why NaturalNeighbours are cool! 
+
+To access the methods easily, you should run `using NearestNeighbours`.
+
+See the [NaturalNeighbours documentation](https://github.com/DanielVandH/NaturalNeighbours.jl) for more details.
+
+This method is fully configurable and will forward all arguments to the relevant NaturalNeighbours.jl functions.
+To understand what the methods do, see the documentation for [`NaturalNeighbours.jl`](https://github.com/DanielVandH/NaturalNeighbours.jl).
+
+## Keyword arguments
+
+The main keyword argument to look at here is `method`, which is the method
+to use for the interpolation.  The other keyword arguments are simply forwarded 
+to NaturalNeighbours.jl's interpolator.
+
+$(Makie.DocStringExtensions.FIELDS)
+"""
+@with_kw struct NaturalNeighboursMethod <: Interpolator
+    "The method to use for the interpolation.  Defaults to `Sibson(1)."
+    method::NaturalNeighbours.AbstractInterpolator = NaturalNeighbours.Sibson{1}()
+    "Whether to use multithreading when interpolating.  Defaults to `nothing`, meaning that triangulation is multithreaded but interpolation is not.  May otherwise be `true` or `false`."
```

Comment:
Judging from `parallel = isnothing(alg.parallel) ? true : alg.parallel` below, it seems that the default is actually to use multithreading for the interpolant. Also note that there is no multithreading for triangulating. So it should probably be

```julia
"Whether to use multithreading for evaluating the interpolant. Defaults to `true`.
parallel::Bool = true
```

Reply:
Thanks for catching that!  I probably confused myself while writing the code.  Will change that option accordingly.